3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+include(FetchContent)
+
+add_subdirectory(dirent-for-Visual-Studio)
+add_subdirectory(EndianPortable)
+add_subdirectory(Getopt-for-Visual-Studio)
+add_subdirectory(LightPcapNg)
+add_subdirectory(hash-library)
+add_subdirectory(MemPlumber/MemPlumber)
+
+if(PCAPPP_INSTALL)
+  install(
+    TARGETS light_pcapng
+    EXPORT PcapPlusPlusTargets
+    LIBRARY
+    ARCHIVE DESTINATION ${PCAPP_INSTALL_LIBDIR}
+    LIBRARY DESTINATION ${PCAPP_INSTALL_LIBDIR}
+    PUBLIC_HEADER DESTINATION ${PCAPP_INSTALL_INCLUDEDIR})
+endif()
+
+if(PCAPPP_BUILD_LIBPCAP)
+  FetchContent_Declare(
+    pcap
+    # ~~~
+    # URL https://www.tcpdump.org/release/libpcap-1.10.1.tar.gz
+    # URL ${CMAKE_CURRENT_SOURCE_DIR}/libpcap-1.10.1.tar.gz
+    # URL_HASH MD5=28e17495004036567c2cc884b51eba45
+    # ~~~
+    GIT_REPOSITORY https://github.com/clementperon/libpcap.git
+    GIT_TAG ce30044e01e99fb2139bf05c0d51be745959f1de
+)
```

Comment:
I don't think we should install or build libpcap, package managers are meant for that...

Reply:
Yes agree, this is something that I found in lot of cmake project but I'm more and more dubious about it. The lib is then internal to the project.

If we keep it external so we should also remove libzstd no ?

Reply:
yes, I commented about it here: https://github.com/seladb/PcapPlusPlus/pull/652#discussion_r928098805